src/tcgwars/logic/impl/gen7/SunMoonPromos.groovy

```diff
@@ -215,8 +215,17 @@ public enum SunMoonPromos implements LogicCardInfo {
   BULBASAUR_SM198("Bulbasaur", 198, Rarity.PROMO, [POKEMON,POKEMON,_GRASS_,BASIC]),
   PSYDUCK_SM199("Psyduck", 199, Rarity.PROMO, [POKEMON,POKEMON,_WATER_,BASIC]),
   SNUBBULL_SM200("Snubbull", 200, Rarity.PROMO, [POKEMON,POKEMON,_FAIRY_,BASIC]),
+  RESHIRAM_CHARIZARD_GX_SM201("Reshiram & Charizard-GX", 201, Rarity.PROMO, [POKEMON, BASIC, POKEMON_GX, TAG_TEAM, _FIRE_]),
+  AMOONGUSS_SM202("Amoongus", 202, Rarity.Promo, [POKEMON,POKEMON,_GRASS_,STAGE1,EVOLUTION]),
+  TAPU_FINI_SM203("Tapu Fini", 203, Rarity.Promo, [POKEMON,POKEMON,_WATER_,BASIC]),
+  NECROZMA_SM204("Necrozma", 204, Rarity.Promo, [POKEMON,POKEMON,_PSYCHIC_,BASIC]),
+  TERRAKION_SM205("Terrakion", 205, Rarity.Promo, [POKEMON,POKEMON,_FIGHTING_,BASIC]),
+  PIKACHU_SM206("Pikachu", 206, Rarity.Promo, [POKEMON,POKEMON,_LIGHTNING_,BASIC]),
+  SUDOWOODO_SM207("Sudowoodo", 207, Rarity.Promo, [POKEMON,POKEMON,_FIGHTING,BASIC]),
+  PORYGON_Z_GX_SM216("Porygon-Z-GX", 216, Rarity.PROMO, [POKEMON, STAGE2, EVOLUTION, POKEMON_GX, _COLORLESS_]), 
```

Comment:
Correct form would be `Rarity.PROMO` (applies to others as well)

Reply:
oops. my bad